add -lrt link flag in order to build on CentOS 6
--- CMakeLists.txt.orig	2019-05-01 07:22:19.000000000 +0800
+++ CMakeLists.txt	2019-08-27 10:22:15.000000000 +0800
@@ -356,6 +356,8 @@
   else()
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s")
   endif()
+else()
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lrt")
 endif()
 
 if(NOT APPLE AND UNIX)
